ubuntu_server_triage.py

```diff
@@ -132,13 +164,15 @@ def main(start, end=None):
 
     connect_launchpad()
     logging.info('Ubuntu Server Bug List')
-    bugs = create_bug_list(start, end)
-    print_bugs(bugs)
+    report_current_backlog(lpname)
+    bugs = create_bug_list(start, end, lpname, bugsubscriber)
+    print_bugs(bugs, open_in_browser)
 
 
 if __name__ == '__main__':
     PARSER = argparse.ArgumentParser()
     PARSER.add_argument('start_date',
+                        nargs='?',
```

Comment:
While I see your case I'd think that this isn't a tool for my grandma to use.
So I'm kind of leaning to "ok to take looooong if not with the right parms".

But, OTOH it is very easy to add a guard.
Now if run without any parms it will warn you, stopping with:
ValueError: No date set, please specify -a if you really want to search without date filter
If you add a -a it will run as before
Searching all bugs, no date filter
[...]

That should be good - If you want to make the no-parm a "auto search yesterday" I'd say feel free to do so if you really think it is needed.